wagtailio/images/migrations/0013_wagtail_42.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.18 on 2023-03-09 18:37
```

Comment:
If I understand https://docs.wagtail.org/en/stable/releases/4.2.html#wagtail-specific-image-field-wagtailimagefield, there won’t be any testing for me to do here as long as the migration runs (it does). So moving on.